src/tmpfiles/tmpfiles.c

```diff
@@ -1091,12 +1108,14 @@ static int parse_xattrs_from_arg(Item *i) {
 }
 
 static int fd_set_xattrs(
+                Context *c,
                 Item *i,
                 int fd,
                 const char *path,
                 const struct stat *st,
                 CreationMode creation) {
 
+        assert(c);
```

Comment:
`c` is not used here?

Reply:
The prototype is what it is because we want to pass it as function param to glob_item_recursively().

The assert() here is of course redundant, but all asserts are

Reply:
ah ok, had missed that